src/Simplex/Messaging/Agent/Client.hs

```diff
@@ -1905,6 +1916,54 @@ withNextSrv c userId usedSrvs initUsed action = do
     writeTVar usedSrvs $! used'
   action srvAuth
 
+-- - currently used servers - those that have state
+-- - previously used servers - have stats but no state - they could be used earlier in session,
+--   or in previous sessions and stats for them were restored
+-- - proxied destination servers (onlyProxiedSMPServers) - based on smpProxiedRelays (key) minus smpClients;
+--   don't have state and stats? since: state is based on smpClients and activeSubs/pendingSubs,
+--   and stats are used to track real network activity (probably also via smpClients),
+--   so they [stats] would be accounted for proxy (unless we want double counting messages as sent and proxied)
+data AgentServersSummary = AgentServersSummary
+  { smpServersState :: Map (UserId, SMPServer) SMPServerState,
+    smpServersStatsData :: Map (UserId, SMPServer) AgentSMPServerStatsData,
+    onlyProxiedSMPServers :: [SMPServer],
+    xftpServersState :: Map (UserId, XFTPServer) XFTPServerState,
+    xftpServersStatsData :: Map (UserId, XFTPServer) AgentXFTPServerStatsData
+  }
+  deriving (Show)
+
+data SMPServerState = SMPServerState
+  { sessions :: Int, -- number of active sessions, based on smpClients - SMPClientVar has client
+    sessionErrs :: Int, -- number of sessions with errors, based on smpClients - SMPClientVar has error
```

Comment:
don't think it's needed - it will be reflected in stats. Error in session var means there is no session, how many errors happened should be reflected in stats (and we shouldn't count cached errors here).